### PR TITLE
Add frontend work to display translated transcripts

### PR DESCRIPTION
**Pull request by awong-dev**
Get the codebase and data file ready for handling translated transcripts.

Also start adding Jest unittest + component test framework. Code is getting too complicated to not have tests.

### CODE REVIEW

File: components/TranscriptControl.tsx [hunk 1]
```diff
@@ -7,12 +7,19 @@ type TranscriptControlParams = {
 
 export default function TranscriptControl({onTimeStampSelected, children} : TranscriptControlParams) {
   function handleClick(e): void {
-    if (e.target.tagName !== 'SPAN') {
+    // Search for nearest span.
```
Comment (mverrey) [inline components/TranscriptControl.tsx:10]: tumblr_lkqm0enAZt1qh59n0o1_500

File: components/TranscriptHeader.tsx [hunk 2]
```diff
@@ -9,12 +9,27 @@ type TranscriptHeaderParams = {
 };
 
 export default function TranscriptHeader({category, title, description, videoId} : TranscriptHeaderParams) {
+/*
+  const translationLinks = (
```
Comment (mverrey) [inline components/TranscriptHeader.tsx:13]: Gonna incorporate that language list from before?

File: utilities/transcript.test.ts [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import { makeDiarizedTranscriptsPath } from 'utilities/transcript';
+
+it('renders homepage unchanged', () => {
```
Comment (mverrey) [inline utilities/transcript.test.ts:3]: Exciting! Looks like it uses the same test structure as Jasmine. Gonna have to read up on how it hooks into React/Next components.

**Review by mverrey — APPROVED** [review]

File: components/TranscriptHeader.tsx (hunk 2 above)
Comment (awong-dev) [reply components/TranscriptHeader.tsx:13]: eventually.

These are the top 5 languages in SPS.  

I'm just being lazy here with the comment.

**Pull request merged**